meilisearch-http/src/data.rs

```diff
@@ -66,7 +67,13 @@ impl Data {
 
         let http_payload_size_limit = opt.http_payload_size_limit;
 
-        let db = Arc::new(Database::open_or_create(opt.db_path, db_opt).unwrap());
+        let db = match  Database::open_or_create(opt.db_path, db_opt) {
+            Ok(db) => Arc::new(db),
+            Err(e) => {
+                error!("{}", e);
+                std::process::exit(1);
```

Comment:
I don't think this is the best way do do so, but let's keep this like that for now :)